bump_version

```diff
@@ -45,6 +51,23 @@ echo $old_version
 old_version_sed="$( sed 's/\./\\./g' <<<"${old_version}" )" # Escape the '.' chars
 sed -i "s/${old_version_sed}/${new_version}/g" ${build_file}
 
+# update nuget package version
+for f in ${mediabrowser_common} ${jellyfin_data} ${mediabrowser_controller} ${mediabrowser_model} ${emby_naming}
```

Comment:
It might be better for this just to be a bash array, i.e.

```
jellyfin_subprojects=( MediaBrowser.Common/MediaBrowser.Common.csproj Jellyfin.Data/Jellyfin.Data.csproj ... )
for subproject in ${jellyfin_subprojects[@]}; do
   ...
```

Reply:
Didn't know about that. Will change it